kubernetes/README.md

```diff
@@ -0,0 +1,31 @@
+# Creating a Kubernetes Deployment
+
+## Summary
+
+The application configuration is in the `config.yaml` Kubernetes secret,
+which stores it in the secret `issecret`.
+
+The deployment created in `deployment.yaml` loads the image from
+quay.io/coreos/issue-sync, maps the `issecret` keys to environment
```

Comment:
Again, this is describing basic kubernetes functionality; this is an implementation detail outside the scope of issue-sync.